Formula/shiboken.rb

```diff
@@ -15,13 +15,13 @@ class Shiboken < Formula
     sha256 "81a1a53a3abdc3b21f5716f30ca1e3be063821f746e7e390ab51e97d45de29c6" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
-  depends_on "qt"
-
   # don't use depends_on :python because then bottles install Homebrew's python
   option "without-python", "Build without python 2 support"
+
   depends_on :python => :recommended if MacOS.version <= :snow_leopard
   depends_on :python3 => :optional
+  depends_on "cmake" => :build
```

Comment:
build options should go before all other options (only exception is `depends_on "foo" => :build" if build.with? "bar"` has to go below `depends_on "bar"` even if `bar` isn't `=> :build`)